gslsys-mfnlin/src/lib.rs

```diff
@@ -0,0 +1,361 @@
+//
```

Comment:
Why not making this part of the main `sys` crate?

Reply:
I made it into a separate create because you have gsl-* in .gitignore.  I went ahead and removed gslsys-mfnlin from the repo.